frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -147,7 +147,7 @@ const PolicyPage = ({
     error: storedPolicyError,
   } = useQuery<IStoredPolicyResponse, Error, IPolicy>(
     ["policy", policyId],
-    () => globalPoliciesAPI.load(policyId as number), // Note: Team members have access to policies through global API
+    () => globalPoliciesAPI.load(policyId as number), // Note: Team users have access to policies through global API
```

Comment:
I know this isn't new, but why are we casting here?

Reply:
idk, this is out of the scope of this PR